server/src/controllers/pollController.ts

```diff
@@ -108,17 +109,15 @@ async function getStudents(courseCode: string, startTime: Date, endTime: Date) {
   }
 }
 
-async function getPollStatus(pollId: any) {
-  if (pollId === null || pollId === undefined || typeof pollId !== "string")
-    return { status: 400, data: { message: "Invalid poll Id" } };
+async function getPollStatus(pollId: string) {
+  if (!pollId) return { status: 400, data: { message: "Invalid poll Id" } };
```

Comment:
If `pollId` is never undefined then we don't really need this if statement right?